src/ItspServices.pServer.Persistence.Sqlite/Repositories/SqlUserRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Data.Common;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("ItspServices.pServer.ServerTest")]
+namespace ItspServices.pServer.Persistence.Sqlite.Repositories
+{
+    class SqlUserRepository
```

Comment:
I would prefix everything with Sqlite instead of Sql or omit it entirely. Sql can also mean MySQL or MSSQL.